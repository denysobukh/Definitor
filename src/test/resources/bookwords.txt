diversify
dabble
profound
yearning
transcend
haunt
majesty 
hold on to
overarching
disguise
being acted
attained
Inconsistency
disproportionately
integrity
inducing
liberate
cognizant
vagaries 
derive
contention
particularly
allegation
thereto
prompt
flaw
suffices 
stale 
Successor  
succeed
predecessor 
contend 
insight 
incurring 
incorporate
interlock
sustain 
heuristics 
spatial
temporal 
caveat
adjacent
sparse 
prolific
bipartite
diligent 
intractable 
augment 
evict
coalesce
stride
virtually
Strip mining
Divergent
smush
replenish
interrogate

