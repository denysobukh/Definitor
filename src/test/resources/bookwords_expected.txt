diversify    ● to start to include more different types or things (Usage: Millions of years ago, changes in the earth's climate caused animal and plant life to diversify.)

dabble       ● to take a slight and not very serious interest in a subject, or try a particular activity for a short period (Usage: He first dabbled in politics when he was at law school.)

profound     ● felt or experienced very strongly or in an extreme way (Usage: His mother's death when he was aged six had a very profound effect on him.)

yearning     ● a strong feeling of wishing for something, especially something that you cannot have or get easily (Usage: I suppose it's because I live in a city that I have this yearning for open spaces.)

transcend    ● to go further, rise above, or be more important or better than something, especially a limit (Usage: The best films are those which transcend national or cultural barriers.)

haunt        ● to cause repeated suffering or anxiety (Usage: Fighting in Vietnam was an experience that would haunt him for the rest of his life.)

majesty      ● If something has majesty, it causes admiration and respect for its beauty (Usage: This music has majesty, power, and passion.)

hold on to   ● to keep something or someone that is important or useful (Usage: I held onto the tech shares after the stock market fell because I knew they would recover.)

overarching  ● most important, because of including or affecting all other areas (Usage: a grand overarching strategy)

disguise     ● to give a new appearance to a person or thing, especially in order to hide its true form (Usage: He disguised himself by wearing a false beard.)


attained     ● past simple and past participle of attain

inconsistency ● a situation in which two things do not match and are opposed (Usage: [ C ] There appear to be inconsistencies in her alibi.)

disproportionately ● too large or too small in comparison to something else, or not deserving its importance or influence (Usage: There are a disproportionate number of girls in the class.)

integrity    ● the quality of being honest and having strong moral principles that you refuse to change (Usage: No one doubted that the president was a man of the highest integrity.)

inducing     ● present participle of induce

liberate     ● to help someone or something to be free (Usage: They said they sent troops in to liberate the people/the country from a dictator.)

cognizant    ● understanding or realizing something (Usage: We should be cognizant of the fact that every complaint is not a justified complaint.)

vagaries     ● unexpected events or changes that cannot be controlled and can influence a situation (Usage: The success of the event will be determined by the vagaries of the weather.)

derive       ● to get something from something else (Usage: The institute derives all its money from foreign investments.)

contention   ● the disagreement that results from opposing arguments (Usage: There's a lot of contention about that issue - for every person firmly in favour, there's someone fiercely against it.)

particularly ● especially, or more than usual (Usage: We're particularly interested to hear from people who speak two or more European languages.)

allegation   ● a statement, made without giving proof, that someone has done something wrong or illegal (Usage: Several of her patients have made allegations of professional misconduct about/against her.)

thereto      ● to the thing just mentioned (Usage: The government has the right to inspect any such establishment or any papers or records relating thereto.)

prompt       ● to make something happen (Usage: The bishop's speech has prompted an angry response from both political parties.)

flaw         ● a fault, mistake, or weakness, especially one that happens while something is being planned or made, or that causes something not to be perfect (Usage: I returned the material because it had a flaw in it.)

suffices     ● to be enough (Usage: I'm taking $400 - I think that should suffice.)

stale        ● no longer new or fresh, usually as a result of being kept for too long (Usage: The bread/biscuits/cake had gone stale.)

successor    ● someone or something that comes after another person or thing (Usage: The university is seeking a successor to its vice chancellor, who retires this spring.)

succeed      ● If you succeed, you achieve something that you have been aiming for, and if a plan or piece of work succeeds, it has the results that you wanted (Usage: She's been trying to pass her driving test for six years and she's finally succeeded.)

predecessor  ● someone who had a job or a position before someone else, or something that comes before another thing in time or in a series (Usage: My predecessor worked in this job for twelve years.)

contend      ● to compete in order to win something (Usage: There are three world-class tennis players contending for this title.)

insight      ● (the ability to have) a clear, deep, and sometimes sudden understanding of a complicated problem or situation (Usage: It was an interesting book, full of fascinating insights into human relationships.)

incurring    ● present participle of incur formal

incorporate  ● to include something as part of something larger (Usage: Suggestions from the survey have been incorporated into/in the final design.)

interlock    ● to fit together firmly (Usage: The edges interlock to form a tight seal.)

sustain      ● to cause or allow something to continue for a period of time (Usage: The economy looks set to sustain its growth into next year.)

heuristics   ● a way of solving problems by discovering things yourself and learning from your own experiences

spatial      ● relating to the position, area, and size of things (Usage: This task is designed to test children's spatial awareness (= their understanding of where things are in relation to other things).)

temporal     ● relating to practical matters or physical things, rather than spiritual ones

caveat       ● a warning to consider something before taking any more action, or a statement that limits a more general statement (Usage: He agreed to the interview, with the caveat that he could approve the final article.)

adjacent     ● very near, next to, or touching (Usage: They work in adjacent buildings.)

sparse       ● small in numbers or amount, often spread over a large area (Usage: a sparse population/audience)

prolific     ● producing a great number or amount of something (Usage: He was probably the most prolific songwriter of his generation.)

bipartite    ● involving two people or organizations, or existing in two parts (Usage: There was a bipartite meeting on border security.)

diligent     ● careful and using a lot of effort (Usage: a diligent student)

intractable  ● very difficult or impossible to control, manage, or solve (Usage: We are facing an intractable problem.)

augment      ● to increase the size or value of something by adding something to it (Usage: He would have to find work to augment his income.)

evict        ● to force someone to leave somewhere (Usage: Tenants who fall behind in their rent risk being evicted.)

coalesce     ● If two or more things coalesce, they come or grow together to form one thing or system.

stride       ● an important positive development (Usage: The West made impressive strides in improving energy efficiency after the huge rises in oil prices during the 70s.)

virtually    ● almost (Usage: Their twins are virtually identical.)

strip mining ● a method of obtaining substances such as coal from the ground that involves removing the top layer of soil instead of digging deep holes underground

divergent    ● different from each other (Usage: a divergent approach/attitude)


replenish    ● to fill something up again (Usage: Food stocks were replenished by/with imports from abroad.)

interrogate  ● to ask someone a lot of questions for a long time in order to get information, sometimes using threats or violence (Usage: Thousands of dissidents have been interrogated or imprisoned in recent weeks.)


